README.rst

```diff
@@ -1,52 +1,65 @@
 Tunneldigger
 ============
 
+L2TPv3 VPN tunneling solution
+
 .. image:: https://travis-ci.org/wlanslovenija/tunneldigger.svg?branch=master
     :target: https://travis-ci.org/wlanslovenija/tunneldigger
 
+About
+-----
+
 Tunneldigger is one of the projects of `wlan slovenija`_ open wireless network.
 It is a simple VPN tunneling solution based on L2TPv3 tunnels supported in
 recent Linux kernels.
 
 .. _wlan slovenija: https://wlan-si.net
 
-Documentation is found at:
+It consists of a client and server portion and is optimized to run on embedded
```

Comment:
Only the client is really optimized for that, I'd say.

This might also be a good opportunity to introduce the "broker" terminology, like

> consists of a client and a server/"broker" portion

Reply:
Fixed.